### DISCUSSION TITLE
Welcome to ProjectsHut 💜🚀 Discussions!

### DISCUSSION THREAD

**Discussion by priyankarpal** [priyankarpal/projectshut#6] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ujjawaltyagii** [comment]
-> If there is any new issue to work on then do let me know !!

**Reply by priyankarpal** [reply]
You can check this issue #40

**Comment by balaji-sivasakthi** [comment]
@priyankarpal Could you assign any new issue under GSSOC'23?

**Reply by priyankarpal** [reply]
> @priyankarpal Could you assign any new issue under GSSOC'23?

just have patience we will assign

**Reply by balaji-sivasakthi** [reply]
Ok

**Comment by highonsemicolon** [comment]
Why there is no database, as there are many PRs just to insert data?

**Reply by priyankarpal** [reply]
we will add backend later

**Comment by alokmishra70** [comment]
Assign me any issue I am a newbie in this openSource Contribution World

**Reply by priyankarpal** [reply]
check opened issues if you're comfortable you can comment for assign

**Comment by balaji-sivasakthi** [comment]
@priyankarpal I'm interested in working on the search bar for the project page. Could you assign it to me? Issue: #647

**Reply by priyankarpal** [reply]
> @priyankarpal I'm interested in working on the search bar for the project page. Could you assign it to me? Issue: #647

try to avoid ping here. you can comment on the issue when it will be labelled as `status: ready for dev`

**Reply by balaji-sivasakthi** [reply]
Ok

**Comment by Hareesh108** [comment]
@priyankarpal Hey bro. I would like to solve below tssue could you assign me?

[Feat]: add skeleton screen animation to the site #907

**Reply by priyankarpal** [reply]
> @priyankarpal Hey bro. I would like to solve below tssue could you assign me?
> 
> [Feat]: add skeleton screen animation to the site #907 
> 

this issue is not ready yet you will see `status:ready for dev` label when it will be ready

**Reply by Hareesh108** [reply]
okay, Thanks!!

**Comment by govardhan-26** [comment]
I have done 2 PRs in this project...But I cant see my name in the leaderboard.. can u please check it ?

**Reply by govardhan-26** [reply]
@priyankarpal those 2 are project additions...wont it be considered for GSSOC'23 level 1 contributions..?

**Reply by priyankarpal** [reply]
added

**Comment by vivekrawat21** [comment]
Can a non-GSSoC contributor add their project?

**Reply by priyankarpal** [reply]
yeah
